packages/buidler-core/src/internal/solidity/compilation-job.ts

```diff
@@ -22,23 +22,23 @@ const SOLC_BUG_9573_VERSIONS = "*";
 export interface CompilationJobsCreationResult {
   jobs: taskTypes.CompilationJob[];
   errors: {
-    [compilationJobCreationError: number]: string[];
+    [compilationJobCreationError: string]: string[];
```

Comment:
Can we use the enum instead? Or would `TASK_COMPILE_SOLIDITY_GET_COMPILATION_JOBS_FAILURES_MESSAGE` break?